docs/hellonode.md

```diff
@@ -128,77 +128,120 @@ It’s now time to deploy your own containerized application to the Kubernetes c
 
 ```shell
-kubectl expose rc hello-node --type="LoadBalancer"
+kubectl expose deployment hello-node --type="LoadBalancer"
 ```
 
-The flag used in this command specifies that we’ll be using the load-balancer provided by the underlying infrastructure (in this case the [Compute Engine load balancer](https://cloud.google.com/compute/docs/load-balancing/)). The `rc` refers to the Kubernetes "replication controller" -- which is a Kubernetes service which controls load balancing and scaling behavior for your cluster. 
+The flag used in this command specifies that we’ll be using the load-balancer provided by the underlying infrastructure (in this case the [Compute Engine load balancer](https://cloud.google.com/compute/docs/load-balancing/)). Note that we expose the deployment, and not the pod directly.  This will cause the resulting service to load balance traffic across all pods managed by the deployment (in this case only 1 pod, but we will add more replicas later).
 
 The Kubernetes master creates the load balancer and related Compute Engine forwarding rules, target pools, and firewall rules to make the service fully accessible from outside of Google Cloud Platform.
 
-To find the publicly-accessible IP address, ask `kubectl` to describe the `hello-node` cluster service:
+To find the ip addresses associated with the service run:
+
+```shell
+kubectl get services hello-node
+NAME         CLUSTER_IP    EXTERNAL_IP     PORT(S)    SELECTOR         AGE
+hello-node   10.3.246.12                   8080/TCP   run=hello-node   23s
+```
+
+The `EXTERNAL_IP` may take several minutes to become available and visible.  If the `EXTERNAL_IP` is missing, wait a few minutes and try again.
 
 ```shell
 kubectl get services hello-node
 NAME         CLUSTER_IP    EXTERNAL_IP     PORT(S)    SELECTOR         AGE
-hello-node   10.3.246.12   23.251.159.72   8080/TCP   run=hello-node   53s
+hello-node   10.3.246.12   23.251.159.72   8080/TCP   run=hello-node   2m
 ```
 
-Note there are 2 IP addresses listed, both serving port 8080. One is the internal IP that is only visible inside your cloud virtual network; the other is the external load-balanced IP. In this example, the external IP address is 23.251.159.72. Traffic to the load-balanced IP will be load balanced to the three nodes you provisioned when initially creating the cluster!
+Note there are 2 IP addresses listed, both serving port 8080.  `CLUSTER_IP` is only visible inside your cloud virtual network.  `EXTERNAL_IP` is externally accessible.  In this example, the external IP address is 23.251.159.72.
 
-You should now be able to reach the service by pointing your browser to this address: http://<EXTERNAL_IP>**:8080**
+You should now be able to reach the service by pointing your browser to this address: http://<EXTERNAL_IP>**:8080** or running `curl http://<EXTERNAL_IP>:8080`
```

Comment:
The first `<EXTERNAL_IP>` was escaped and `** **` wasn't rendered as bold text on the website. 
<img width="649" alt="screen shot 2016-03-24 at 2 18 20 pm" src="https://cloud.githubusercontent.com/assets/4876867/14031702/4c9ddf96-f1cb-11e5-9660-a0fd71d4baca.png">